### ISSUE TITLE
More meaningful charging state

### ISSUE THREAD

**Issue by kifort** [thehookup/MQTT-Roomba-ESP01#12]
Wrong packet ID is used for battery voltage at
https://github.com/thehookup/MQTT-Roomba-ESP01/blob/7ad7cd09321a43e0b74d5bf430566a7bcb32051b/RoombaESP01_CONFIGURE.ino#L126

Packet ID 21 is the Charging State
Packet ID 22 is the Voltage

It is only a variable naming issue, but the integer value of Packet ID 21 can be mapped to a more meaningful name based on the Code and Charging State mapping.

**Comment by pilot1981** [comment]
I agree with you! You can suggest a pullrequest to this code?  I don’t think @thehookup can review its code
